docs/using-the-compiler.rst

```diff
@@ -199,6 +199,13 @@ Output Description
             // The Ethereum Contract ABI. If empty, it is represented as an empty array.
             // See https://github.com/ethereum/wiki/wiki/Ethereum-Contract-ABI
             abi: [],
+            // See the Metadata Output documentation (serialised JSON string)
+            metadata: "{...}",
+            // User documentation (natspec)
+            userdoc: {},
+            // Developer documentation (natspec)
+            devdoc: {},
+            // EVM-related outputs
```

Comment:
(two lines below): The idea was that the IR is not too EVM specific, but perhaps we can keep it here and later add an additional one higher up. In that IR we might want to use some functions which are not part of the IR itself but will be added when we know the target machine.